components/message/demo/close.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { NzMessageService } from 'ng-zorro-antd';
+
+@Component({
+  selector: 'nz-demo-message-close',
+  template: `
+    <button nz-button [nzType]="'default'" (click)="createBasicMessage()">Customized close callback</button>
+  `,
+  styles: []
+})
+export class NzDemoMessageCloseComponent implements OnInit {
+
+  ngOnInit(): void {
+    this.message.nzAfterClose.subscribe(message => {
```

Comment:
If you listen to `NzMessageService`'s event, you have to manually manage the subscription. In another word, unsubscribe this subscription when the component is destroyed. So, this API because verbose for users.